embassy-nrf/src/pwm.rs

```diff
@@ -815,6 +831,38 @@ impl<'d, T: Instance> SimplePwm<'d, T> {
         let max_duty = self.max_duty() as u32;
         clk / max_duty
     }
+
+    /// Sets the PWM-Channel0 output drive strength
+    #[inline(always)]
+    pub fn set_ch0_drive(&self, drive: OutputDrive) {
+        if let Some(pin) = &self.ch0 {
+            pin.conf().write(|w| w.drive().variant(convert_drive(drive)));
```

Comment:
these should be .modify() so it leaves the other fields unchanged.

Reply:
Oh of course thanks, I wrongly assumed `write()` doing the job for me, I'm a newbie in embedded-rust world as you can tell 😄